### PR TITLE
Rewrite

### PR DESCRIPTION
**Pull request by willem4ever**
Added example mosquitto_pub with TLS  + output example for an activation

### CODE REVIEW

File: Backend/Connect/Application.md [hunk 1]
```diff
@@ -88,14 +92,48 @@ With the Mosquitto MQTT client you would connect and schedule a downlink with th
...
 #### Activations
 
 Path: `<AppEUI>/devices/<DevEUI>/activations` (use `+/devices/+/activations` to get activations from all devices)
 
-> TODO: Example
+topic returned on activation 
+````
+<EUI>/devices/0004A30B001B2F9C/activations
+```
+payload contains metatdata only
```
Comment (johanstokking) [inline Backend/Connect/Application.md]: Typo

File: Backend/Connect/Application.md [hunk 2]
```diff
@@ -64,9 +64,13 @@ The application payload `payload` is base64 encoded.
 
 With the Mosquitto MQTT client you would connect and subscribe with the following command:
 
-```
+````
```
Comment (johanstokking) [inline Backend/Connect/Application.md]: Please be consistent with number of back quotes

File: Backend/Connect/Application.md [hunk 3]
```diff
@@ -64,9 +64,13 @@ The application payload `payload` is base64 encoded.
...
 mosquitto_sub -h staging.thethingsnetwork.org -t '0807060504030201/devices/+/up' -u 0807060504030201 -P 'I0f+e1W+CWgIiuIC4SjR5cpLxFZQfK2agDEpuCBpttI='
-```
+````
+To enable TLS you would use the following command:
+````
+mosquitto_sub --cafile <path>/mqtt-ca.pem -p 8883 -h staging.thethingsnetwork.org -t '0807060504030201/devices/+/up' -u 0807060504030201 -P 'I0f+e1W+CWgIiuIC4SjR5cpLxFZQfK2agDEpuCBpttI='
```
Comment (johanstokking) [inline Backend/Connect/Application.md:72]: This is not specific to your PR, but for Windows we need double quotes

File: Backend/Connect/Application.md [hunk 4]
```diff
@@ -88,14 +92,48 @@ With the Mosquitto MQTT client you would connect and schedule a downlink with th
...
+
 
 **Note**: the `port` is not yet supported, and will always be `1`.
 
 #### Activations
 
 Path: `<AppEUI>/devices/<DevEUI>/activations` (use `+/devices/+/activations` to get activations from all devices)
 
-> TODO: Example
+topic returned on activation
```
Comment (johanstokking) [inline Backend/Connect/Application.md]: Make sentence

File: Backend/Connect/Application.md (hunk 3 above)
Comment (willem4ever) [inline Backend/Connect/Application.md:72]: Single quotes are used for all the other examples ....

File: Backend/Connect/Application.md (hunk 1 above)
Comment (willem4ever) [inline Backend/Connect/Application.md]: fixed

File: Backend/Connect/Application.md (hunk 4 above)
Comment (willem4ever) [inline Backend/Connect/Application.md]: Ok

File: Backend/Connect/Application.md (hunk 2 above)
Comment (willem4ever) [inline Backend/Connect/Application.md]: changed

**Comment by willem4ever** [comment]
Made the changes and pushed them, anything else I need to do ?

**Comment by htdvisser** [comment]
Thanks @willem4ever!

**Pull request merged**